app/lib/theme/widgets/FieldLabel.tsx

```diff
@@ -21,7 +21,14 @@ const FieldLabel: React.FC<Props> = ({
   ) {
     return null;
   }
-  return <label htmlFor={htmlFor}>{getRequiredLabel(label, required)}</label>;
+  return (
+    <>
+      <label htmlFor={htmlFor}>{getRequiredLabel(label, required)}</label>
+      {uiSchema && uiSchema["bcgov:help-text"] && (
+        <small>&nbsp;{uiSchema["bcgov:help-text"]}</small>
```

Comment:
This should be rendered by the `FieldTemplate`, but I guess the widgets rendering the label isn't making it possible right now. This should be refactored in #328

Reply:
Aaah I hadn't seen #328, and I started doing it in this PR